src/content/guides/code-splitting.md

```diff
@@ -126,6 +131,20 @@ __webpack.config.js__
       index: './src/index.js',
       another: './src/another-module.js'
     },
++  module: {
++     rules: [
++       {
++         test: /\.js$/,
++         exclude: /node_modules/,
++         ues: {
++           loader: 'babel-loader',
```

Comment:
Can you please keep consistency in object keys quotes? Some have double, some have single.

Reply:
:+1: